### ISSUE TITLE
Maybe widgets like Apple Books in future?

### ISSUE THREAD

**Issue by TheFermi0n** [Aidoku/Aidoku#264]
In future maybe Aidoku can add widgets like Apple Books as shown in the screenshots below. It will make it much easier to access currently reading manga. Now that we will be having interactive widgets, it will make it much easier to access currently reading.

|Size|Preview|Remarks|
|---|---|---|
|small|![small](https://github.com/Aidoku/Aidoku/assets/63984668/634ac7b5-a30c-4fed-bbef-bafb8c684f7f)|Instead of completion percentage we can have the current volume, chapter and page number for Aidoku|
|medium|![medium](https://github.com/Aidoku/Aidoku/assets/63984668/c7097b91-38f1-4c3f-82b0-9cf969a5108e)|Instead of completion percentage we can have the current chapter and page number for Aidoku|
|large|![large](https://github.com/Aidoku/Aidoku/assets/63984668/9dbb8518-624f-4a7a-ba54-b7fb2eb3be4f)| I don't think daily goal is necessary for Aidoku as we mostly use a tracker to track our manga progress. But it won't be a bad addition to Aidoku itself.|

PS. color can be changed to either match the manga itself or the Aidoku accent color.

**Comment by Skittyblock** [comment]
yeah, this is a good idea and something that I'd like to look into for fun since I haven't really messed with widgets much
